### PR TITLE
Add sanity check of index devId before accessing array

### PR DESCRIPTION
**Pull request by lealem47**

### CODE REVIEW

File: wolfcrypt/src/port/intel/quickassist.c [hunk 1]
```diff
@@ -557,6 +557,12 @@ int IntelQaOpen(WC_ASYNC_DEV* dev, int devId)
         return ASYNC_INIT_E;
     }
 
+    if (devId >= g_numInstances) {
+        printf("devId %d out of range. Number of instances=%u.\n", devId,
```
Comment (dgarske) [inline wolfcrypt/src/port/intel/quickassist.c:561]: Please use: `fprintf(stderr, "IntelQA: devId %d exceeds number of instances %u\n", devId, g_numInstances);`

**Review by dgarske — CHANGES_REQUESTED** [review]

File: wolfcrypt/src/port/intel/quickassist.c (hunk 1 above)
Comment (lealem47) [reply wolfcrypt/src/port/intel/quickassist.c]: Done

File: wolfcrypt/src/port/intel/quickassist.c [hunk 2]
```diff
@@ -557,6 +557,12 @@ int IntelQaOpen(WC_ASYNC_DEV* dev, int devId)
         return ASYNC_INIT_E;
     }
 
+    if (devId >= g_numInstances) {
+        fprintf(stderr, "devId %d out of range. Number of instances=%u\n",
```
Comment (dgarske) [inline wolfcrypt/src/port/intel/quickassist.c:561]: Please use the specific text I outlined. Thanks

**Review by dgarske — CHANGES_REQUESTED** [review]

File: wolfcrypt/src/port/intel/quickassist.c (hunk 2 above)
Comment (lealem47) [reply wolfcrypt/src/port/intel/quickassist.c]: Done

**Review by dgarske — APPROVED** [review]

**Pull request merged**